schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+   "title": "Instrument Specification Definition",
+   "description": "The Instrument Specification Definition for Astro CSM",
+   "type": "object",
+
+   "properties": {
+       "BORESIGHT": {
+	   "description": "Boresight (x,y,z) vector",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "description": "Component of the boresight vector",
+		 "minItems": 3,
+                 "maxItems": 3
+            }
+       },
+       "DETECTOR_CENTER": {
+           "description": "Center of the detector in pixel coordinates",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "minItems": 2,
+                 "maxItems": 2
+            }
+       },
+       "CENTER_EPHEMERIS_TIME": {
+           "description": "The center ephemeris time of the image",
+           "type": "number"
+       },
+       "STARTING_EPHEMERIS_TIME": {
+           "description": "The starting ephemeris time of the image",
+           "type": "number"
+       },
+       "FOCAL_LENGTH_MODEL": {
+            "description":  "A key to get information about time/temp dependent focal length models",
+            "type": "object",
+            "properties": {
+                "key": {"type": "string"},
+                "focal_length": {"type": "number"},
+                "focal_epsilon":{"type": "number"}
+            },
+   	"required": ["focal_length"]
+       },
+       "IFOV": {
+           "description": "Instantaneous field of view in degrees",
+           "type": "number"
+       },
+       "IMAGE_LINES": {
+           "description": "The number of lines in the image",
+           "type": "number"
+       },
+       "IMAGE_SAMPLES": {
+           "description": "The the number of samples in the image",
+           "type": "number"
+       },
+
+       "RADII": {
+           "description": "The radii of the target.",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "minItems": 1,
+                 "maxItems": 3
+            }
+       },
+       "OPTICAL_DISTORTION": {
+           "description": "Information needed to convert from undistorted to distorted image plane",
+           "type": "object",
+	   "properties" : {
+	      "coefficients": {
+	         "type": "array",
+		 "items": {
+		    "type": "number"
+		  }
+		},
+	        "key" : {
+		    "type": "string",
+		    "description": "key into LUT to determine equation to use"
+		    }
+	 },
+	"required": ["coefficients", "equation_key"]
+       },
+
+       "STARTING_DETECTOR_SAMPLE": {
+           "description": "The sample on the detector where the data starts",
+           "type": "number"
+       },
+       "STARTING_DETECTOR_LINE": {
+           "description": "The line on the detector where the data starts",
+           "type": "number"
+       },
+       "FOCAL2PIXEL_X": {
+           "description": "Transformation x coefficients from focal plane (mm) to pixel coordinates",
```

Comment:
This converts to detector pixels, it should be mentioned in the description because it is not clear if it converts from focal to detector pixels or focal to image pixels.

Reply:
:+1:

Reply:
Also, we should note the coefficient order is constant, x , y.